packages/web-api-send-notification-runner/src/tasks/notification-sender-web-api-client.ts

```diff
@@ -34,7 +34,7 @@ export class NotificationSenderWebAPIClient {
         const requestBody = {
             scanId: notificationSenderConfigData.scanId,
             runStatus: notificationSenderConfigData.runStatus,
-            scanStatus: notificationSenderConfigData.scanStatus,
+            scanStatus: notificationSenderConfigData.runStatus === 'failed' ? null : notificationSenderConfigData.scanStatus,
```

Comment:
use isEmpty(notificationSenderConfigData.scanStatus) ? undefined : notificationSenderConfigData.scanStatus

Also, in runner set the scanStatus to undefined instead of null.